### ISSUE TITLE
Yarn 2 support

### ISSUE THREAD

**Issue by darkship** [AnomalyInnovations/serverless-bundle#143]
I tried to use serveless-bundle with yarn 2. Unfortunally I get the following error :
```
➤ YN0000: Error: serverless-bundle tried to access resolve, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
➤ YN0000: 
➤ YN0000: Required package: resolve (via "resolve")
➤ YN0000: Required by: serverless-bundle@npm:1.9.1 ...
```

Yarn 2 needs all the dependencies to be listed in your package.json and it appears that `resolve` isn't.
`resolve` is used in https://github.com/AnomalyInnovations/serverless-bundle/blob/c955b72b3216d4fa2e64a9a4b86bd4dbe1fd9473/scripts/test.js#L39

**Comment by jayair** [comment]
Hmm so if we include `resolve` in the dependencies it would fix this? It looks like we should be adding it anyway.

Can you add it to the dependencies and submit a PR?

**Comment by francisu** [comment]
I'm using version 3.2.1 of the serverless-bundle plugin (you release a lot of versions quickly, I love that) and the "resolve" issue is not there, but there is an issue with the "import-fresh" dependency. It should be moved from devDependencies to dependencies. One that's done, it works fine with yarn 2. If you want I will do a PR.

**Comment by darkship** [comment]
@francisu I wasn't at the point where I could deploy a function yet. It happened when I run `yarn test`

**Comment by jayair** [comment]
@francisu Yup that looks like a mistake on our side. Please open a PR (if you haven't already, I haven't checked all my notifications yet).

**Comment by jayair** [comment]
Nvm, I'm working on it now. I'll put the `import-fresh` change in. I'll put out a new release shortly. Then you can test it with Yarn 2 and let me know.
